stdlib/public/Platform/glibc.modulemap.gyb

```diff
@@ -41,7 +41,122 @@ module SwiftGlibc [system] {
   link "execinfo"
 % end
 
-  header "SwiftGlibc.h"
+% if CMAKE_SDK not in ["ANDROID"]:
+  header "stdc-predef.h"
+  header "features.h"
+% end
+
+  // C standard library
+  header "complex.h"
+  header "ctype.h"
+  header "errno.h"
+  header "fenv.h"
+  header "float.h"
+  header "inttypes.h"
+  header "iso646.h"
+% if CMAKE_SDK not in ["LINUX", "ANDROID"]:
+  header "libutil.h"
+% end
+  header "limits.h"
+  header "locale.h"
+  header "math.h"
+  header "pty.h"
+  header "setjmp.h"
+  header "signal.h"
+  header "stdarg.h"
+  header "stdbool.h"
+  header "stddef.h"
+  header "stdint.h"
+  header "stdio.h"
+  header "stdlib.h"
+  header "string.h"
+  header "tgmath.h"
+  header "time.h"
+% if CMAKE_SDK in ["OPENBSD"]:
+  header "util.h"
```

Comment:
@3405691582 added this for his OpenBSD port, and neither glibc nor Bionic have it.